airbyte-cdk/python/airbyte_cdk/sources/file_based/file_types/unstructured_parser.py

```diff
@@ -79,14 +81,18 @@ def parse_records(
         logger: logging.Logger,
         discovered_schema: Optional[Mapping[str, SchemaType]],
     ) -> Iterable[Dict[str, Any]]:
+        format = _extract_format(config)
         with stream_reader.open_file(file, self.file_read_mode, None, logger) as file_handle:
-            markdown = self._read_file(file_handle, file)
+            markdown = self._read_file(file_handle, file, format, logger)
+            if markdown is None:
```

Comment:
Nit: this can be more simply put with 
```
if markdown is not None:
    yield {
        "content": markdown,
        "document_key": file.uri,
    }
```
instead of having the `yield from` & `return`.